README.md

```diff
@@ -121,10 +121,8 @@ color:
   when: auto
   # How to colorize the output.
   # When "classic" is set, this is set to "no-color".
-  # Possible values: default, <theme-file-name>
-  # when specifying <theme-file-name>, lsd will look up theme file
-  # XDG Base Directory if relative, e.g. ~/.config/lsd/themes/<theme-file-name>.yaml,
-  # The file path if absolute
+  # Possible values: default, custom
+  # When "custom" is set, lsd will look in the config directory for `colors.yaml`.
```

Comment:
I have reconsidered the logic, how about we drop the custom logic, and detect the `colors.yaml` directly, if the colors file is found, we use it as a custom theme.

so that we don't have to update the colors theme file and both this configuration, it seems a little bit duplicated

Reply:
Can you please clarify what you mean by that? I don't see the issue of the code duplication here, since one is deprecated anyway and should be phased out at some point.

Reply:
sorry for not explaining myself.

I was saying that we should drop the `custom` option for the theme option because we could detect whether the `colors.yaml` file existed.

it's not about codes, it is about usage

Reply:
hi @peppidesu, did you have some time to work on this PR again?

let me explain myself more clear,
1. we deprecated all the theme configuration
2. we detect the existence of `colors.yaml` file every run
3. if `colors.yaml` existed, we use it directly
4. if not, we fallback to default

WDYT?

Reply:
In that case we should do the same for icons, since it has a field for default/custom.

Reply:
Hmmm...make sense, let's leave those to later improvement